### ISSUE TITLE
Detections that should be filtered out are present in taghits_daily

### ISSUE THREAD

**Issue by j-sayers** [MotusDev/Motus-TO-DO#1294]
There appear to be entries in `taghits_daily` which should be filtered out. 

- There are 3500 daily records for Lotek tags with runLen < 4. 
- About 1000 daily records of CTT tags with runLen == 1 
- 6500 records of tag_id 0 (mfg_id '999999'), with about 15 of these having runLen < 4. 
- Maybe just a misunderstanding of mine but I thought that detections without tag deployments shouldn't be present in this table. That's not the case (4.7 million records) but maybe that's by design and they are filtered out after

**Comment by denislepage** [comment]
It is possible for a daily summary to have the hits straddling different days or tag or receiver deployment. I checked, and it explains at least some (probably all) of those cases, including cases when len = 1. The daily summary is calculated for any combination of tag and/or deploy ID. In one example, a single hit happened to be before the start of the receiver deployment on a given day, but the rest of them were included. All hits that day were assigned to the same tag deployment, so you have 2 daily summary records (one where recvDeployID = 0 (with nhits = 1), and another where recvDeployID = 2102 (nhits = 5041).

These detections were part of the same run, which should in theory have a len > 3, so I am not sure we should be filtering out the single hits that fall just outside the boundaries like this.

That said, the scripts had a condition for only including runs that had len > 3, but it is commented out at the moment. @cjardine-bsc  may need to dig into this.

**Comment by j-sayers** [comment]
Makes sense that hits belong to a longer run would be included if they are split between days, even if it's just one hit (though the case could also be made for not including that single hit...)

But is run length not calculated at the device level, irrespective of user-entered receiver deployment metadata? 

Saying that, I just answered my own question. It's not run length reported in taghits_daily, but the number of hits during that day. Usually they match, but occasionally they won't.

**Comment by j-sayers** [comment]
Does anyone know what tag_id 0 (mfg_id '999999') is? There are a fair number of hits of that "tag" in taghits_daily as well.
